docs/intro/manifesto.md

```diff
@@ -0,0 +1,76 @@
+[Cred](../concepts/cred.md) based on how it was connected to other
+contributions, and how much Cred those other contributions earned.
+Mathematically, we do this by constructing a network of contributions called a
+Contribution Graph, and then running the CredRank algorithm on it.
+
+Finally, we acknowledge that in order to be sustainable, communities need to
+interact with the "real world" economy of dollars and cents. For that reason,
+every community can issue [Grain](../concepts/grain.md), a community-specific
+cryptocurrency. Grain is distributed based on Cred, so it is all earned by
+community members and contributors. Sponsors purchase Grain, and then use it
+for [Boosting](../concepts/boosting.md), which allows them to influence the
+Cred scores, and thus the community's priorities. Basically, Sponsors help make
+the community economically viable, and in gratitude the community gives
+sponsors some infuence over the community's goals, values, and priorities.
+
+In the short term, SourceCred is focused on supporting open-source crypto
+projects. Crypto projects are an ideal fit because they need a way to
+distribute the tokens that they natively generate, and need a decentralized,
+"new-world" appraoch for doing so--the old world approaches of corporations and
+shareholder value just don't fit. The key early adopter is the SourceCred
+project itself: we have no corporate structure and no investors, but we reward
+everyone in the community via Cred and Grain. (Big thanks to [Protocol
+Labs](https://protocol.ai/) for being our first and biggest sponsor.) We're
+also thrilled to be partnered with other awesome and idealistic projects like
+[MetaGame](https://metagame.wtf/) and
+[AraCred](https://aracred.github.io/website/).
+
+In the medium term, we want to make the whole open-source ecosystem
+economically sustainable. The core idea of open source is that we can make
+better technology, and empower more people, if we give away our work for free
+rather than encumbering it with licenses and paywalls. This is an incredibly
+powerful idea, but it's held back from its full potential because it depends on
+unpaid and overworked volunteers. Once enough open-source projects are using
+SourceCred, we will create a "MetaCred" instance that flows Cred and Grain
+across the open-source dependency graph. This will ensure that every project in
+the ecosystem--from flashy user-facing projects to the core infrastructure they
+depend on--will share in the success of the ecosystem. The result will be a
+golden age of flourishing open-source.
+
+In the long run, we hope that SourceCred can help unlock big transformations to
+align the world economy with our shared values. So many of the thorny problems
+facing society seem to come down to us valuing the wrong things. We put a price
+on medicines when we should be valuing healthiness. We compensate investment
+bankers when we should be appreciating teachers. We "redevelop" rainforests
```

Comment:
> We compensate investment bankers when we should be appreciating teachers.
This line feels unnecessarily polarizing, and I'd claim there's a more powerful and potentially relevant framing for some of these.  I'd suggest something like:
* We compensate executives based on a company's stock price, not its impact on the world.
* We often compensate teachers based on factors like where they they teach and how long they've taught, rather than how much students learn.
* We value the livestock we can raise on a cleared rain forest, but can't express the value the forest itself provides.

And then I'd love to see this point to a more concrete future direction, like "Markets work best when participants have accurate and robust information.  By making capturing and quantifying new types information SourceCred should provide qualitatively more complete valuations that align with our shared values."

Hopefully this is useful, even if this undergoes a larger rewrite.